---
timestamp: '2025-01-13T13:35:13.637336'
level: INFO
issuer: instance_solver
event_type: metadata
data:
    log_file: /home/remote/u7899572/lifted-white-plan-domain-repair/exp_logs_anu/dfs
        relax-all lp1/data-network-opt18-strips__pp13-err-rate-0-1.yaml
    instance_id: data-network-opt18-strips__pp13-err-rate-0-1
    search_algorithm: dfs
    plan_length: 10
    domain_class: data-network-opt18-strips
    instance_name: pp13-err-rate-0-1
    ground_plan:
    - (load data-0-7 server4 number4 number8 number0 number4)
    - (send data-0-7 server4 server1 number4 number16 number0 number4)
    - (load data-0-9 server2 number4 number8 number0 number4)
    - (send data-0-9 server2 server1 number4 number16 number4 number8)
    - (process data-0-7 data-0-9 data-1-8 script1 server1 number4 number16 number8
        number12)
    - (process data-1-8 data-0-9 data-2-5 script12 server1 number2 number16 number12
        number14)
    - (process data-1-8 data-0-7 data-2-2 script7 server1 number2 number16 number14
        number16)
    - (send data-2-2 server1 server3 number2 number8 number0 number2)
    - (save data-2-2 number2 server3)
    - (save data-2-5 number2 server1)
    lifted_plan:
    -   - load
        - ?data-0-7
        - ?server4
        - ?number4
        - ?number8
        - ?number0
        - ?number4
    -   - send
        - ?data-0-7
        - ?server4
        - ?server1
        - ?number4
        - ?number16
        - ?number0
        - ?number4
    -   - load
        - ?data-0-9
        - ?server2
        - ?number4
        - ?number8
        - ?number0
        - ?number4
    -   - send
        - ?data-0-9
        - ?server2
        - ?server1
        - ?number4
        - ?number16
        - ?number4
        - ?number8
    -   - process
        - ?data-0-7
        - ?data-0-9
        - ?data-1-8
        - ?script1
        - ?server1
        - ?number4
        - ?number16
        - ?number8
        - ?number12
    -   - process
        - ?data-1-8
        - ?data-0-9
        - ?data-2-5
        - ?script12
        - ?server1
        - ?number2
        - ?number16
        - ?number12
        - ?number14
    -   - process
        - ?data-1-8
        - ?data-0-7
        - ?data-2-2
        - ?script7
        - ?server1
        - ?number2
        - ?number16
        - ?number14
        - ?number16
    -   - send
        - ?data-2-2
        - ?server1
        - ?server3
        - ?number2
        - ?number8
        - ?number0
        - ?number2
    -   - save
        - ?data-2-2
        - ?number2
        - ?server3
    -   - save
        - ?data-2-5
        - ?number2
        - ?server1


---
timestamp: '2025-01-13T13:35:13.642379'
level: INFO
issuer: instance_solver
event_type: ground_repair
data:
    repair_length: 0
    repair: ''


---
timestamp: '2025-01-13T13:47:11.441048'
level: ERROR
issuer: batch_solver
event_type: error
data:
    error_type: MemoryError
    error_message: 'Command ''[''/home/projects/u7899572/conda-envs/main/bin/python3'',
        ''instance_solver.py'', ''dfs'', ''/home/remote/u7899572/lifted-white-plan-domain-repair/input/benchmarks-G1'',
        ''/home/remote/u7899572/lifted-white-plan-domain-repair/exp_logs_anu/dfs relax-all
        lp1/data-network-opt18-strips__pp13-err-rate-0-1.yaml'', ''1000000'', ''data-network-opt18-strips__pp13-err-rate-0-1'',
        ''1.0'', ''unary'']'' died with <Signals.SIGABRT: 6>.'
    stack_trace: "Traceback (most recent call last):\n  File \"batch_solver.py\",\
        \ line 126, in worker\n    result = subprocess.run(cmd, check=True, timeout=params['timeout_seconds'],\n\
        \  File \"/home/projects/u7899572/conda-envs/main/lib/python3.8/subprocess.py\"\
        , line 516, in run\n    raise CalledProcessError(retcode, process.args,\n\
        subprocess.CalledProcessError: Command '['/home/projects/u7899572/conda-envs/main/bin/python3',\
        \ 'instance_solver.py', 'dfs', '/home/remote/u7899572/lifted-white-plan-domain-repair/input/benchmarks-G1',\
        \ '/home/remote/u7899572/lifted-white-plan-domain-repair/exp_logs_anu/dfs\
        \ relax-all lp1/data-network-opt18-strips__pp13-err-rate-0-1.yaml', '1000000',\
        \ 'data-network-opt18-strips__pp13-err-rate-0-1', '1.0', 'unary']' died with\
        \ <Signals.SIGABRT: 6>.\n"
    instance_id: data-network-opt18-strips__pp13-err-rate-0-1
    process_info:
        cmd:
        - /home/projects/u7899572/conda-envs/main/bin/python3
        - instance_solver.py
        - dfs
        - /home/remote/u7899572/lifted-white-plan-domain-repair/input/benchmarks-G1
        - /home/remote/u7899572/lifted-white-plan-domain-repair/exp_logs_anu/dfs relax-all
            lp1/data-network-opt18-strips__pp13-err-rate-0-1.yaml
        - '1000000'
        - data-network-opt18-strips__pp13-err-rate-0-1
        - '1.0'
        - unary
        duration: '719.28'
        python_executable: /home/projects/u7899572/conda-envs/main/bin/python3
    stdout: ''
    stderr: 'terminate called after throwing an instance of ''Minisat22::OutOfMemoryException''

        '


